tutorials/python/lbs/first/first_example.ipynb

```diff
@@ -0,0 +1,438 @@
+    "nodes = []\n",
+    "n_cells = 10\n",
+    "length = 2.0\n",
+    "xmin = - length / 2\n",
+    "dx = length / n_cells\n",
+    "for i in range(n_cells + 1):\n",
+    "    nodes.append(xmin + i * dx)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Orthogonal Mesh Generation\n",
+    "We use the `OrthogonalMeshGenerator` and pass the list of nodes per dimension. Here, we pass 2 times the same list of\n",
+    "nodes to create a 2D geometry with square cells. Thus, we create a square domain, of side length 2, centered on the origin (0,0).\n",
+    "\n",
+    "We also partition the 2D mesh into 2x2 subdomains using `KBAGraphPartitioner`. Since we want the split the x-axis in 2,\n",
+    "we give only 1 value in the xcuts array (x=0). Likewise for ycuts (y=0). The assignment to a partition is done based on where the\n",
+    "cell center is located with respect to the various xcuts, ycuts, and zcuts (a fuzzy logic is applied to avoid issues).\n",
+    "\n",
+    "The resulting mesh and partition is shown below:\n",
+    "\n",
+    "![Mesh_Partition](images/first_example_mesh_partition.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "meshgen = OrthogonalMeshGenerator(\n",
+    "    node_sets=[nodes, nodes],\n",
+    "    partitioner=KBAGraphPartitioner(\n",
+    "        nx=2,\n",
+    "        ny=2,\n",
+    "        xcuts=[0.0],\n",
+    "        ycuts=[0.0],\n",
+    "    )\n",
+    ")\n",
+    "grid = meshgen.Execute()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Material IDs\n",
+    "When using the in-house `OrthogonalMeshGenerator`, no material IDs are assigned. The user needs to\n",
+    "assign material IDs to all cells. Here, we have a homogeneous domain, so we assign a material ID\n",
+    "with value 0 for each cell in the spatial domain."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "grid.SetUniformBlockID(0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Cross Sections\n",
+    "We load the cross sections from an OpenSn file format.\n",
+    "See the tutorials' section on materials for more details on cross sections."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "xs_matA = MultiGroupXS()\n",
+    "xs_matA.LoadFromOpenSn(\"xs_1g_MatA.xs\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Volumetric Source\n",
+    "We create a volumetric multigroup source which will be assigned to the material with given block IDs.\n",
```

Comment:
concept of material is gone. -->
We create a volumetric multigroup source which will be assigned to cells with given block IDs.